lib/libutee/include/utee_defines.h

```diff
@@ -45,15 +47,30 @@
 #define TEE_CHAIN_MODE_GCM              0x8
 #define TEE_CHAIN_MODE_PKCS1_PSS_MGF1   0x9	/* ??? */
 
-	/* Bits [31:28] */
-#define TEE_ALG_GET_CLASS(algo)         (((algo) >> 28) & 0xF)
 
-#define TEE_ALG_GET_KEY_TYPE(algo, with_private_key) \
-        (TEE_ALG_GET_MAIN_ALG(algo) | \
-            ((with_private_key) ? 0xA1000000 : 0xA0000000))
+static inline uint32_t _tee_alg_get_class(uint32_t algo)
```

Comment:
I'd prefer `__` prefix since `_` is supposed to be file scope only and this inline functions continues to be defined outside this file.

Reply:
Fixed.